doc/api/http.md

```diff
@@ -1390,9 +1387,14 @@ Limits maximum incoming headers count. If set to 0, no limit will be applied.
 
 <!-- YAML
 added: v14.11.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/41263
+    description: The default request timeout changed
+                 from no timeout to 300s (5 minutes).
 -->
 
-* {number} **Default:** `0`
+* {number} **Default:** `300000`
```

Comment:
This seems like a bit of a breaking change? I have cases where I'm ok with having a requests running for hours, i.e. long polling. Which is probably a reasonable use case.

Reply:
Should at minimum be a notable change?

Reply:
AAh, sorry I misunderstood the purpose of the timeout. The naming might be a bit misleading?

Reply:
If we have a headersTimeout maybe this corresponds to a `bodyTimeout`?

Reply:
Also the meaning of what `requestTImeout` means before and after this PR is significantly different (i.e. end of incoming message instead of outgoing) which again might motivate a name change.

Reply:
Ehm... why? The logic of this PR does not change the semantic of the timeout (time to receive the entire request from the client). Why do you think it will change?

Reply:
I agree on this. That's a fair point.
But I think our new default timeout is very large for any legit purpose. 

If the client/server are legitimately slow then the user should know and fine tune accordingly.

Reply:
I'm fine with leaving it as future improvement. Good job!